stm32cube/stm32g4xx/drivers/src/stm32g4xx_ll_utils.c

```diff
@@ -329,6 +329,9 @@ ErrorStatus LL_PLL_ConfigSystemClock_HSI(LL_UTILS_PLLInitTypeDef *UTILS_PLLInitS
     {
       UTILS_ClkInitStruct->AHBCLKDivider = LL_RCC_SYSCLK_DIV_1;
       LL_RCC_SetAHBPrescaler(UTILS_ClkInitStruct->AHBCLKDivider);
+
+      /* Enable PLL and switch system clock to PLL */
+      status = UTILS_EnablePLLAndSwitchSystem(pllfreq, UTILS_ClkInitStruct);
```

Comment:
I agree this fix makes sense. It should be added also to `LL_PLL_ConfigSystemClock_HSE` further down below (lines 420+).